akka-remote/src/main/scala/akka/remote/RemoteWatcher.scala

```diff
@@ -287,15 +287,9 @@ private[akka] class RemoteWatcher(
    * does not exist.
    */
   def reWatch(address: Address): Unit =
```

Comment:
My question was: can we not just unconditionally fire the DeathWatchNotifications (without reWatch) in case a new UID is seen?